app/controllers/api/v1/npq_applications_controller.rb

```diff
@@ -68,6 +68,22 @@ def access_scope
       def npq_application
         @npq_application ||= npq_lead_provider.npq_applications.includes(:cohort, :npq_course, participant_identity: [:user]).find(params[:id])
       end
+
+      def funded_place
+        if FeatureFlag.active?(:npq_capping)
+          accept_permitted_params.dig("attributes", "funded_place")
+        end
+      end
+
+      def accept_permitted_params
+        params.require(:data).permit(:type, attributes: %i[funded_place])
+      rescue ActionController::ParameterMissing => e
```

Comment:
I was testing with this and was thinking we avoid this because of past years where we won't need to pass in this request vs 2024, otherwise providers will get an error. Happy to chat through